knnvector/operations/default.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "warmup-queries",
+    "operation-type": "search",
+    "index": "{{ target_index_name | default('target_index') }}",
+    "param-source": "random-knn-query-param-source",
+    "k": {{ query_k  | default(100) }},
+    "dimension": {{ target_index_dimension | default(512) }},
+    "field": "{{ target_field_name | default('target_field') }}",
+    "min_value": {{ query_min_value  | default(-10.0) }},
+    "max_value": {{ query_max_value  | default(10.0) }},
+    "include-in-results_publishing": false
+},
+{
+    "name": "force-merge",
+    "operation-type": "force-merge",
+    "request-timeout": {{ target_index_force_merge_timeout | default(7200) }},
+    "index": "{{ target_index_name | default('target_index') }}",
+    "mode": "polling",
+{%- if target_index_max_num_segments is defined %}
+    "max-num-segments": {{ target_index_max_num_segments }},
+{%- endif %}
```

Comment:
lets use a default value as 1 for this.

Reply:
What's the reasoning behind defaulting to 1 segment? Does that give better performance result?

Reply:
yes, single segment leads to best performance for the vector engine as segments are searched sequentially.

Reply:
Ack